ui/main/src/app/modules/admin/components/editmodal/users/edit-user-modal.component.ts

```diff
@@ -76,6 +76,8 @@ export class EditUserModalComponent implements OnInit {
             comment: new FormControl('', []),
             groups: new FormControl([]),
             entities: new FormControl([]),
+            //NOSONAR : The regex is used to validate the input of the authorizedIPAddresses field
+            // and is not a security issue as it is done on the client side
             authorizedIPAddresses: new FormControl('', [Validators.pattern(/^((\d+(\.\d+){3}),?\s?)+$/)])
```

Comment:
Change the regex pattern to 
^(?:\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3},?)+$
The nested check ( {3} ) is what causes the sonar trigger.

Reply:
You are right , done